apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -3665,6 +3671,49 @@ static ItemComponent mapEobCommonItemRevenueStatusCode(
     return item;
   }
 
+  /**
+   * Gets the reference variable
+   *
+   * @param ccwCodebookVariable the {@ CcwCodebookVariable} to get the url
+   * @return url as a string
+   */
+  static String getReferenceUrl(CcwCodebookVariable ccwCodebookVariable) {
+    return CCWUtils.calculateVariableReferenceUrl(ccwCodebookVariable);
+  }
+
+  /**
+   * Checks to see if there is a extension that already exists in the careteamcomponent so a
+   * duplicate entry for extension is not added
+   *
+   * @param careTeamComponent - Careteam component
+   * @param referenceUrl the {@link String} is the reference url to compare
+   * @param codeValue - the {@link String} is the code value to compare
+   * @return {@link Boolean} whether it was found or not
+   */
+  public static boolean careTeamHasMatchingExtension(
+      CareTeamComponent careTeamComponent, String referenceUrl, String codeValue) {
+
+    if (!Strings.isNullOrEmpty(referenceUrl)
+        && !Strings.isNullOrEmpty(codeValue)
+        && careTeamComponent.getExtension().size() > 0) {
+      List<Extension> extensions = careTeamComponent.getExtensionsByUrl(referenceUrl);
+
+      for (Extension ext : extensions) {
+        Coding coding = null;
```

Comment:
codes seems a little awkward; no need to even attempt to create a variable unless it is instanceof Coding).

Reply:
done

Reply:
done